src/main/resources/carpet.mixins.json

```diff
@@ -211,7 +211,9 @@
     "WorldRenderer_scarpetRenderMixin",
 
     "SoundEngineInt_cleanLogsMixin",
-    "SoundSystem_cleanLogsMixin"
+    "SoundSystem_cleanLogsMixin",
+
+    "StructureBlockScreen_structureBlockLimitMixin"
```

Comment:
Isn't this one supposed to be only a client side mixin - that class doesn't exist on the server only.

Reply:
It does be a client side mixin. It's in the "client" array

Reply:
lol - totally missed that - yeah, its good.